### ISSUE TITLE
Make smooth scrolling and Windows Scrolling Personality a setting option (maybe default on)

### ISSUE THREAD

**Issue by Goldman90** [brave/brave-browser#35600]
There's a lot of people asking for "edge like scroll" for brave in reddit and other places. I myself just came back to brave from a short while using edge, and found the scrolling unconfortable, first i though it was because smooth scrolling was off, but after turning it on (in flags, though it should be in settings at this point) i still found it hard to use. Only after discovering  Windows Scrolling Personality and turning that on, i could change browsers confidently.

I know this is my preference and a lot of people might not like it, but i believe, at least, the options should be in settings, so people wont just outright abandon the browser if they actually got used to how scroll works in edge, for example

**Comment by bsclifton** [comment]
Thanks for recommending @Goldman90 - this is a pretty nice feature! For anyone reading this that wants to try it, you can open Brave and visit brave://flags/#windows-scrolling-personality. Enable this and restart - you should be good to go

Interestingly enough, this is available for macOS and Linux too. @rebron we can definitely make a setting for this. Maybe it would make sense to put under brave://settings/system - hopefully that page isn't too cluttered.  I'm curious if we should show for all OS? Or just Windows.

**Comment by rebron** [comment]
cc: @jonathansampson

**Comment by bsclifton** [comment]
OK - @emerick and I got a chance to talk to Gaston Rodriguez on Chromium's Slack. Here's the update he had provided:
> Edge had WSP enabled by default, but it was identified as a source of battery regression (scrolling with WSP enabled drains more battery due to more calculations being needed), and because of this we decided to un-ship the feature. The next step after our un-shipping is done is to start removing the code in the chromium code base, which is planned to start around October

This is w/ regards to brave://flags/#windows-scrolling-personality

Due to this, I think the best action is to close this issue as `wontfix`. Which is a shame - because I turned it on after my last post here and have been enjoying it. ☹️ But seeing as the code will be removed, we'd have to remove any work we do anyways.

There is another flag available:
brave://flags/#smooth-scrolling

I'm not sure of the differences with this enabled/disabled - but we could create a new issue for this specifically.

**Comment by Goldman90** [comment]
Wtf unship? i understand having it disabled by default and keeping it as a flag, but it really sucks to actually remove it, not all pcs use batteries...
sadly smooth scrolling does not feel exactly the same.

**Comment by bsclifton** [comment]
@Goldman90 (and others reading this) - if you wanted to give some feedback to folks directly, I'd encourage you to check out the official Chromium issue here:
https://issues.chromium.org/issues/359747082

You do need to sign up w/ a Google account in order to leave a comment. I'll go ahead and leave a comment myself there now as I really enjoy the feature and I'm not going to turn it off until it "unships" and stops working.

**Comment by l4kr** [comment]
Bring it back.

**Comment by SunsetTechuila** [comment]
> Bring it back.

Nah, ads and AI/crypto slop are more important priorities
